watertap/unit_models/zero_order/tests/test_mabr_zo.py

```diff
@@ -145,16 +145,10 @@ def test_solution(self, model):
             model.fs.unit.properties_treated[0].conc_mass_comp["nitrate"]
         )
 
-        assert pytest.approx(4e-12, rel=1e-2) == value(
+        assert pytest.approx(0, abs=1e-9) == value(
             model.fs.unit.properties_byproduct[0].flow_vol
         )
-        assert pytest.approx(200, rel=1e-5) == value(
-            model.fs.unit.properties_byproduct[0].conc_mass_comp["bod"]
-        )
-        assert pytest.approx(200, rel=1e-5) == value(
-            model.fs.unit.properties_byproduct[0].conc_mass_comp["nitrate"]
-        )
```

Comment:
Why have the two assert statements on lines 151 - 156 been removed?

Reply:
The byproduct stream is zero for this unit, so no need for these composition tests.

Reply:
Are you saying that for `MABRZO`, the byproduct stream always has a flowrate of zero?

Reply:
Yeah, the byproduct stream flowrate is zero based on their provided data.

Reply:
In that case, does it make more sense to build this model as a SISO instead of a SIDO?

Reply:
I think it's better to keep as a SIDO in the case where alternative data is fed into the unit, allowing for some byproduct flowrate.